android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIScrollView.java

```diff
@@ -575,13 +589,28 @@ protected void onScrollChanged(int l, int t, int oldl, int oldt)
 		{
 			super.onScrollChanged(l, t, oldl, oldt);
 			KrollDict data = new KrollDict();
+			KrollDict contentData = new KrollDict();
+			int contentWidth = 0;
+			int contentHeight = 0;
+
 			if (!isScrolling && isTouching) {
 				isScrolling = true;
 				getProxy().fireEvent(TiC.EVENT_DRAGSTART, data);
 			}
+
+			if (getChildCount() > 0) {
+				final View child = getChildAt(0);
+				TiDimension dimW = new TiDimension(child.getWidth(), TiDimension.TYPE_WIDTH);
+				TiDimension dimH = new TiDimension(child.getHeight(), TiDimension.TYPE_HEIGHT);
+				contentWidth = TiConvert.toInt(dimW.getAsDefault(child));
+				contentHeight = TiConvert.toInt(dimH.getAsDefault(child));
+			}
```

Comment:
Same as above. Content width/height should be doubles, not ints.

Reply:
done